### PR TITLE
feat: create push_devices table

### PR DESCRIPTION
**Pull request by alexruzenhack**
### Acceptance Criteria
- create the table `push_devices`.

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: db/migrations/20221108235926-create-pushdevices.js.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    await queryInterface.createTable('push_devices', {
+      device_id: {
+        type: Sequelize.STRING(256),
+        allowNull: false,
+        primaryKey: true,
+      },
+      push_provider: {
+        type: Sequelize.ENUM(['ios', 'android']),
+        allowNull: false,
+        defaultValue: 'android',
```
Comment (tuliomir) [inline db/migrations/20221108235926-create-pushdevices.js.js]: I don't think we should have a default value here. If some code tries to register a device without specifying the provider, I think it's better to throw an error than to suppose a value.

What do you think about this?

**Review by tuliomir — CHANGES_REQUESTED** [review]
I think we should also update the [`/DATABASE.md` file](https://github.com/HathorNetwork/hathor-wallet-service/blob/master/DATABASE.md) with the table's description.

It's a very helpful consolidated reference for understanding the database as a whole.

File: db/migrations/20221108235926-create-pushdevices.js.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        defaultValue: false,
+      },
+      updated_at: {
+        type: 'TIMESTAMP',
+        allowNull: false,
+        defaultValue: Sequelize.literal('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'),
+      },
+    });
+  },
+  async down(queryInterface, Sequelize) {
```
Comment (andreabadesso) [inline db/migrations/20221108235926-create-pushdevices.js.js]:
```suggestion
  async down(queryInterface) {
```

**Review by andreabadesso — CHANGES_REQUESTED** [review]

**Comment by alexruzenhack** [comment]
Task of https://github.com/HathorNetwork/hathor-wallet-service/issues/300

**Review by tuliomir — APPROVED** [review]

**Review by andreabadesso — APPROVED** [review]

**Pull request merged**